sqlscripts/local/0001-jaeger-index.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE IF NOT EXISTS %s (
      INDEX idx_tags tags TYPE bloom_filter(0.01) GRANULARITY 64,
      INDEX idx_duration durationUs TYPE minmax GRANULARITY 1
 ) ENGINE MergeTree()
+%s
```

Comment:
Would it make more sense to have the following? 

```
TTL timestamp + INTERVAL %s DAY DELETE
```

Reply:
It's needed for case with no TTL